src/main/java/Controller.java

```diff
@@ -1,19 +1,25 @@
 public class Controller {
-    private final Game game;
+    private Game game;
+    private final GameFactory gameFactory;
     private final Display display;
-
-    public Controller(Game game, Display display) {
-        this.game = game;
+    
+    public Controller(GameFactory gameFactory, Display display) {
+        this.gameFactory = gameFactory;
         this.display = display;
     }
-
+    
     public void playGame() {
+        game = createGame();
```

Comment:
Could you inline this method?

Reply:
Good question. I'll look into this. Thanks, Ashley